src/psyclone/tests/psyGen_test.py

```diff
@@ -183,6 +183,83 @@ def apply(self, _1):
     assert trans.name == "TestTrans"
 
 
+def test_transformation_get_options():
+    ''' Test that the get_option method behaves in the
+    expected way.'''
+    class TestTrans(Transformation):
```

Comment:
You could move this class definition out of this test and into the parent module. Then it's available to be reused in each test, instead of having to define it multiple times.

Reply:
Each TestTrans is different in the arguments it has I think to test different sections of the code.